### DISCUSSION TITLE
Key prices not being updated properly?

### DISCUSSION THREAD

**Discussion by lordvague** [TF2Autobot/tf2autobot#963] [Q&A]
I have 3 bots.  I've seen that prices using the standard API lagging behind quite a bit from average prices from time to time.

Today I noticed that one bot had current average price for keys, yet my other two bots that do a lot more trading were lagging behind several hours.  I attempted to pricecheck and ask for rates, but they would not update their price and remained well behind the recent  fluctuation while the one bot remains currently priced.  

Why is this?  Can I do anything to make them more current or is this just a problem with the current pricing API?

**Comment by lordvague** [comment]
2+ hours later, one more bot is now current, still one final bot price is lagging from over ~8 hours ago.

**Comment by idinium96** [comment]
Hello, sorry for the very late response. If you run all the bots on the same machine, it might be a problem with the Websocket connection. Try to run the bots on a separate machine, for example on a VPS.
